bin/9l

```diff
@@ -31,6 +31,7 @@ case "$tag" in
 	;;
 *Darwin*)
 	ld="${CC9:-gcc} -m64 $CC9FLAGS"
+	extralibs="-lresolv"
```

Comment:
Does this cause any unintended side effects on earlier versions of macOS?

Reply:
I'm not sure, I don't have a machine running an older version

Reply:
It looks like `-lresolv` has been in use as far back as OS X 10.6, to fix this exact linker error, see [here](https://stackoverflow.com/a/1204470).

Reply:
@bhuntsman is this sufficient?